content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure)
+files, uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to dvc via `dvc add` or
+`dvc commit` or `dvc repro` but the metafiles are not committed to git yet. The
+second change shows the new changes to the data that is not tracked by DVC yet
+or what you could `dvc commit`. The third change shows the files that are
+missing from the cache. Some of those could be pulled from the remote.
+
+Using `--untracked-files`, it'll show the files that are not being tracked by
+DVC or Git.
+
+By default, `dvc data status` shows changes in the level the file/directory is
+being tracked in. With `--granular`, it will show changes in file-level.
+
+The `--unchanged` option will list all of the files that aren't listed in
+committed changes or uncommitted changes.
+
+### Output
+
+The `dvc data status` displays changes in multiple categories such as:
```

Comment:
I think we need a table with explanations for this. May be a diagram, not sure. But w/o any explanations it's very hard to understand what this command does.

Reply:
I am trying to avoid any table. Diagram may be abstract. I think we should try to put it in simple words what those changes are, and follow up with examples.

Reply:
I like the simplicity, but I think we should then put an example right here in the description. Or at least be explicit - "to understand the meaning check the example". Example should cover all of them then (?) and should be easier to understand. I'm not sure it's easy to achieve with throwing at people an output from the project that they haven't seen and didn't touch any files in it.

Reply:
Reading the whole description again. The section name is output, but there is a list with name, no example to see. And to be honest, in this form it doesn't give enough value to exist (I mean the Output section). It should just embedded into the second paragraph above. It describes the same in some abstract fashion (which makes it hard to read). I think it's better instead of the half of the text about and the whole this section show a simplified / schematic snippet with an output and explain sections briefly underneath it.